Packs/CommonDashboards/Scripts/UseCaseAdoptionMetrics/README.md

```diff
@@ -0,0 +1,20 @@
+This dashboard is meant to provide a high level overview on the usage of XSOAR. It contains datapoints and metrics on enriched and observed threat intelligence, playbook and automation executions, and data sources being ingested into xsoar. It is useful for determining automation maturity and overall return on investment. 
```

Comment:
```suggestion
This dashboard is meant to provide a high level overview on the usage of Cortex XSOAR. It contains datapoints and metrics on enriched and observed threat intelligence, playbook and automation executions, and data sources being ingested into Cortex XSOAR. It is useful for determining automation maturity and overall return on investment. 
```
But this is a script, not a dashboard. I don't think the description is correct.